source/patterns/@aws-solutions-constructs/core/lib/utils.ts

```diff
@@ -123,3 +124,15 @@ export function generateResourceName(
 function removeNonAlphanumeric(s: string) {
   return s.replace(/[^A-Za-z0-9]/g, '');
 }
+
+/**
```

Comment:
This should be in the test folder, not production code (I'm assuming from the name it's only called by tests?)

we already have core/test/test-helper.ts